diff --git a/include/RE/G/GHashSetBase.h b/include/RE/G/GHashSetBase.h
index 159c3a1..fd31118 100644
--- a/include/RE/G/GHashSetBase.h
+++ b/include/RE/G/GHashSetBase.h
@@ -211,7 +211,7 @@ namespace RE
 			Clear();
 			if (a_src.IsEmpty() == false) {
 				SetCapacity(a_memAddr, a_src.GetSize());
-				for (const_iterator it = a_src.Begin(); it != a_src.End(); ++it) {
+				for (const_iterator it = a_src.begin(); it != a_src.end(); ++it) {
 					Add(a_memAddr, *it);
 				}
 			}
@@ -361,7 +361,7 @@ namespace RE
 
 		[[nodiscard]] UPInt GetSize() const
 		{
-			return table ? 0 : (UPInt)table->entryCount;
+			return table ? static_cast<UPInt>(table->entryCount) : 0;
 		}
 
 		void CheckExpand(void* a_memAddr)
 
